lib/functions.php

```diff
@@ -414,7 +414,7 @@ function elgg_solr_get_adapter_options() {
 function elgg_solr_has_settings() {
 	$host = elgg_get_plugin_setting('host', 'elgg_solr');
 	$port = elgg_get_plugin_setting('port', 'elgg_solr');
-	$path = elgg_get_plugin_setting('path', 'elgg_solr');
+	$path = elgg_get_plugin_setting('solr_path', 'elgg_solr');
```

Comment:
nope, not sure why that's changed, but it fixes the open issue for 2.1.1. the loading was halted because it couldn't find the port.